### ISSUE TITLE
 I lost access to the micom test, even after creating the mcmtest_activate.ini and mcmtest_activate_4medianav.ini files.

### ISSUE THREAD

**Issue by Joaobars** [m-a-x-s-e-e-l-i-g/MediaNav-to-Evolution-Upgrade#3]
Hi, I recently updated my Media Nav from version 4.0.1 to version 7.0.5 MD that you provided on GitHub. Everything went well, but I lost access to the micom test, even after creating the mcmtest_activate.ini and mcmtest_activate_4medianav.ini files. When I press the micom button, the radio reboots, and I need to enter the security code. Can you help me with this issue?

**Comment by m-a-x-s-e-e-l-i-g** [comment]
Have you tried my code generator?
https://renault-dacia-radio-code-generator.netlify.app/

You can read about it here
 https://github.com/m-a-x-s-e-e-l-i-g/MediaNav-to-Evolution-Upgrade/blob/main/Radio_Code.md

**Comment by Ruben7173** [comment]
> Have you tried my code generator? https://renault-dacia-radio-code-generator.netlify.app/
> 
> You can read about it here https://github.com/m-a-x-s-e-e-l-i-g/MediaNav-to-Evolution-Upgrade/blob/main/Radio_Code.md

I don't think this is what he meant as I am having the same issue. I added the files on my pen drive to load dev menu and having the option micom, when I select it it simply turns the radio off and when I turn in on it asks for the radio code which is fine as I know it. But I need to access the micom and it doesnt seem to work

**Comment by m-a-x-s-e-e-l-i-g** [comment]
- Try downgrading to version 4.0.3, as some users have reported similar issues resolved with earlier firmware.
- Try adding empty `mcmtest_activate.ini` and/or `mcmtest_activate_4medianav.ini` files in the root folder on a FAT32 USB drive.

Please let me know if you were able to resolve this as it might help others :)

**Comment by Mr-Pingu** [comment]
https://www.youtube.com/watch?v=-3xIEfK35rw You could try this

**Comment by m-a-x-s-e-e-l-i-g** [comment]
The right code was supposed to be `0362` but the MICOM menu is not opening...



I'll try the following solutions soon:
## 🔧 Method 1: `testmode.key` File on USB

Quickest and safest method to trigger Micom or Factory menu.

### Steps:

1. Format a USB stick to **FAT32**
2. Create an **empty file** named exactly: testmode.key
3. Plug it into the USB port while the MediaNav is powered on
4. Wait **5–20 seconds**
5. One of the following should happen:
- The system boots into **Micom Test Mode**
- A **developer menu** appears
- **Nothing happens** — try restarting the system with the USB still inserted

---

## 🧪 Method 2: Use a dboot-compatible Tool (MicomTestTool, FactoryMenuLauncher)

If `testmode.key` doesn’t work:

### Steps:

1. Download a dboot tool like:
- **MicomTestTool**
- **FactoryMenuLauncher**
- **Medianav Unlock Pack**

2. Extract it to the **root** of the USB stick (maintain folder structure)
3. Plug it into the system and **reboot**
4. A custom menu interface will appear with access to:
- **Micom settings**
- **Feature toggles**
- **Debug/testing tools**
